src/main/java/edu/usf/cutr/grha/gtfs/GtfsConstants.java

```diff
@@ -0,0 +1,16 @@
+package edu.usf.cutr.grha.gtfs;
+
+public class GtfsConstants {
```

Comment:
If these are only used in the `TncToGtfsWriter`, let's just move them to the top of that class and make them private.